apps/web/src/hooks/useExperimentalFeatureEnabled.ts

```diff
@@ -0,0 +1,12 @@
+import { EXPERIMENTAL_FEATURES } from 'config/experminetalFeatures'
+import Cookies from 'js-cookie'
+import { useState } from 'react'
+
+const useExperimentalFeatureEnabled = (featureFlag: EXPERIMENTAL_FEATURES) => {
+  const isCookieInFeatureFlags = () => Cookies.get(`ctx-${featureFlag}`)?.toString() === 'true'
+  const [cookieValue] = useState<boolean>(isCookieInFeatureFlags())
```

Comment:
the initial state should be empty. And let's use `useEffect` to get the cookie value on mount.

Reason: `js-cookie` only works on browser env and this hook may be running on server end